src/scripts/persist-indicators/index.js

```diff
@@ -15,15 +15,21 @@ fs.readFile('config.json', 'utf8', function(err, conf) {
  * Class in charge of loading autoindicators turning them into datavalues.
  */
 function AutoIndicatorsLoader(conf) {
+
+	//get apiversion
+	var apiversion="";
+	console
+	if (typeof(conf.apiversion)!="undefined" && conf.apiversion!="") apiversion = "/"+conf.apiversion; 
+	
     //used endpoints
     this.endpoints = {
-        INDICATORGROUPS: "/24/indicatorGroups.json?filter=name:eq:AutoIndicators&fields=indicators[*]",
-        ATTRIBUTE_BY_NAME: "/24/attributes?fields=id&filter=name:eq:",
-        DATASETS: "/24/dataSets/UID.json?fields=periodType",
-        ANALYTICS: "/24/analytics.json?displayProperty=NAME&skipMeta=true&",
-        DATAVALUESETS: "/24/dataValueSets" 
+        INDICATORGROUPS: apiversion+"/indicatorGroups.json?filter=name:eq:AutoIndicators&fields=indicators[*]",
+        ATTRIBUTE_BY_NAME: apiversion+"/attributes?fields=id&filter=name:eq:",
+        DATASETS: apiversion+"/dataSets/UID.json?fields=periodType",
+        ANALYTICS: apiversion+"/analytics.json?displayProperty=NAME&skipMeta=true&",
+        DATAVALUESETS: apiversion+"/dataValueSets"
     };
-
+    
     //rest config
```

Comment:
This is a little bit picky but try to avoid indentation and blank lines if they are not needed. For big PR it makes everything a little bit messy and difficult to follow